### PR TITLE
docs(root): updated Get Started pages

### PR DESCRIPTION
**Pull request by mi6-378**
Prettier run, nav priority altered, Overview pages created

## Summary of the changes

Created new PR linking to old one here: #1209

### CODE REVIEW

**Review by GCHQ-Developer-530 — CHANGES_REQUESTED** [review]
Looking good - just one thing I noticed across all the get started pages is that the cards aren't responsive to smaller screen sizes (they're causing horizontal scroll).

Could you update the cards to be displaying in a column if they're on a small screen?

Screenshot 2024-11-22 at 10 59 30

**Review by GCHQ-Developer-530 — DISMISSED** [review]

File: src/content/structured/get-started/images/designing-for-accessibility.svg
Comment (GCHQ-Developer-112) [inline src/content/structured/get-started/images/designing-for-accessibility.svg:1]: Please get rid of duplicate images, ideally keep the 2x sized images and get ride of the smaller ones

**Review by GCHQ-Developer-112 — COMMENTED** [review]
There's inconsistency with how the horizontal cards look on mobile (375px). To avoid truncation, there should be a small viewport size chosen where no image is shown
Screenshot 2024-11-26 at 11 13 34

**Review by GCHQ-Developer-299 — APPROVED** [review]

**Comment by GCHQ-Developer-112** [comment]
One of the images is causing an x-overflow (horizontal scroll)
Screenshot 2024-12-03 at 14 29 10

**Review by GCHQ-Developer-530 — APPROVED** [review]

**Review by GCHQ-Developer-530 — APPROVED** [review]

**Review by GCHQ-Developer-299 — DISMISSED** [review]

File: src/templates/Homepage/index.css [hunk 1]
```diff
@@ -112,7 +112,7 @@
   }
 }
 
-@media screen and (min-width: 769px) and (max-width: 992px) {
```
Comment (GCHQ-Developer-299) [inline src/templates/Homepage/index.css:115]: This 992px breakpoint shouldn't be changed, i don't think - it's based on our breakpoint css tokens
```css
  --ic-breakpoint-xs: 0;
  --ic-breakpoint-sm: 576px;
  --ic-breakpoint-md: 768px;
  --ic-breakpoint-lg: 992px;
  --ic-breakpoint-xl: 1200px;
```

Comment (GCHQ-Developer-299) [reply src/templates/Homepage/index.css:115]: if you need it to be bigger, maybe use 1200px?

**Review by GCHQ-Developer-299 — DISMISSED** [review]

File: src/config.js [hunk 2]
```diff
@@ -123,9 +123,13 @@ module.exports = {
         "With a focus on accessibility and ease of use, these components equip developers to create responsive and inclusive experiences.",
       ],
       button: [
```
Comment (GCHQ-Developer-530) [inline src/config.js:125]: The href's further up in this file for the design section need updating as well

**Review by GCHQ-Developer-530 — DISMISSED** [review]

**Review by GCHQ-Developer-299 — DISMISSED** [review]

**Review by GCHQ-Developer-299 — APPROVED** [review]

**Review by GCHQ-Developer-530 — APPROVED** [review]

**Pull request merged**